scripts/setup.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh -ex
+export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+export DEBIAN_FRONTEND=noninteractive
+export ETCKEEPER_NO_PUSH=1
+
+die() {
+    echo "$@" >&2
+    exit 1
+}
+
+in_docker() {
+    [ -f /.dockerenv ]
+}
+
+# Some basic assumptions
+apt update
+apt install -y apt-transport-https etckeeper
+
+if [ -d /etc-git/ ]; then
+    [ -d /etc-git/.git ] || die "/etc-git is not a git repo, something has gone wrong"
+else
+    # Assume we are pulling the current production branch
+    git clone https://github.com/hashbang/shell-etc.git /etc-git
+fi
+
+
+# Fix some things that maintainer scripts expect
+# Those should probably be filed as Debian bugs...
+mkdir -p /var/lib/bitlbee
+mkdir -p /usr/local/lib/luarocks/rocks-5.1 #WTF is that even needed?
```

Comment:
The question is more whether the package installs properly without it (that was the issue with `bitlbee` too)